zipkin/src/main/java/zipkin2/Component.java

```diff
@@ -38,6 +38,11 @@ public CheckResult check() {
     return CheckResult.OK;
   }
 
+  public String name() {
```

Comment:
please revert this method as it is adding a public method to a base type that existed for a long time, for a single call site. We take public methods on base types more seriously here vs other projects.

Reply:
I'm okay with this change. Just want to note that the name reported by the health endpoint will change for some components. E.g "InMemoryStorage" will become "InMemoryStorage{traceCount=123}"